Tom-Kelly-Answers.md

```diff
@@ -0,0 +1,14 @@
+Q1: So you're finishing up your time in this CSCC program. Why did you decide to embark on this journey?
+* *Tom:* I studied Computer Science at Ohio State, but I graduated nearly 8 years ago. I wanted to get a refresher on some of the new topics, as I have not been coding in quite a few years.
+
+Q2: I love to travel. Tell me a bit about your favorite travel destination so far. What's on your travel bucket list?
+* *Tom:* I traveled to Ireland last October, It was beautiful. I met some distant relatives in Ballycroy, County Mayo.
+![Ballycroy](/images/BallyCroy.jpg)
+* *Tom:* I want to travel to see the Pyramids at Giza.
+
+Q3: Name your favorite member of The Beatles, and your favorite Beatles song. You do know at least one Beatles song, right? If you are a Beatles fan, feel free to give me your top 5 or so.
+* *Tom:* Paul McCartney -- Revolution. I'm more of a vaporwave kinda guy, check out [Home - Resonance](https://www.youtube.com/watch?v=8GW6sLrK40k)
+
+Q4: Do you have any pets? Species/Name/Details please. Murray wants to know... (he's not as smart as that picture makes him seem).
+* *Tom:* I have a cat, Little One. She likes to lie by the heater and paw at birds in the morning, as soon as they start chirping outside. Her favorite place to do so is the window at the head of my bed. 
```

Comment:
Hilarious!  Murray also loves barking at things he sees from the window... big scary things like leaves.  :-)